docs/sdk/push/migration/xamarin-android.md

```diff
@@ -24,7 +24,7 @@ If you are using App Center Build, you must make sure Mono version is 5.8 or hig
 
 ## 2. AndroidManifest.xml
 
-Edit the project's **AndroidManifest.xml** file, then insert the following lines **inside** the <application> section:
+Edit the project's **AndroidManifest.xml** file and remove the following lines in <application> section, if present:
```

Comment:
Could we make the wording similar to the original? Instead of `in <application> section`, I'd suggest do the same `**inside** the <application> section`. I think `inside` is a little more clear than `in`.

Reply:
Good suggestion